pubspec.yaml

```diff
@@ -1,21 +1,22 @@
 name: surf_widget_test_composer
-description: A new Flutter package project.
+description: Utility that simplifies widget and golden testing.
```

Comment:
this library only for golden testing, not so?
I guess we should change package name

Reply:
fixed